samples-generator/lib/createBuildingsTile.js

```diff
@@ -74,10 +86,11 @@ function createBuildingsTile(options) {
             var batchTableExtra = generateBatchTableExtra(buildings);
             batchTableJson = combine(batchTableJson, batchTableExtra);
         }
+
         if (createBatchTableBinary) {
-            var batchTableJsonAndBinary = generateBatchTableBinary(buildings);
-            batchTableJson = combine(batchTableJson, batchTableJsonAndBinary.json);
+            batchTableJsonAndBinary = (use3dTilesNext) ? generateBatchTableBinary3dTilesNext(buildings) : generateBatchTableBinary(buildings);
```

Comment:
```suggestion
            batchTableJsonAndBinary = use3dTilesNext ? generateBatchTableBinary3dTilesNext(buildings) : generateBatchTableBinary(buildings);
```

Reply:
fixed this and every other instance of a ternary condition missing parens, check with the regex: `=\s*([^(].*[^)]) \? (.*) : (.*)`
It's not a perfect regex, it breaks on === etc, but if you use it in the `search` tab in vscode you can preview everywhere that `(=|==|=== cond ? a : b)` occurs